doc/Support/SNMP-Configuration-Examples.md

```diff
@@ -0,0 +1,150 @@
+source: Support/SNMP-Configuration-Examples.md
+
+# SNMP configuration examples
+
+Table of Content:
+- [Devices](#devices)
+    - [Cisco](#cisco)
+        - [Adaptive Security Appliance (ASA)](#adaptive-security-appliance-asa)
+        - [IOS](#ios)
+        - [IOS](#ios)
+        - [IOS XE](#ios-xe)
+        - [NX-OS](#nx-os)
+        - [Wireless LAN Controller (WLC)](#wireless-lan-controller-wlc)
+    - [Infoblox](#infoblox)
+        - [NIOS 7.x](#nios-7x)
+    - [Juniper](#juniper)
+        - [Junos OS](#junos-os)
+    - [Palo Alto](#palo-alto)
+        - [PANOS 6.x/7.x](#panos-6x7x)
+- [Operating systems](#operating-systems)
+    - [Linux (snmpd)](#linux-snmpd)
+    - [Windows Server 2012 R2](#windows-server-2012-r2)
+
+## Devices
+
+### Cisco
+#### Adaptive Security Appliance (ASA)
+1. Launch ASDM and connect to your device
+2. Go to Configuration > Management Access > SNMP
+3. Add your community string
+4. Add in the "SNMP Host Access List" section your LibreNMS server IP address
+5. Click Apply and Save
+
+#### IOS
+
+``` 
+snmp-server community YOUR-COMMUNITY RO
+snmp-server contact YOUR-CONTACT
+snmp-server location YOUR-LOCATION
+```
+
+#### IOS XE
+``` 
+snmp-server community YOUR-COMMUNITY RO
+snmp-server contact YOUR-CONTACT
+snmp-server location YOUR-LOCATION
+```
+
+#### NX-OS
+``` 
+snmp-server community YOUR-COMMUNITY RO
+snmp-server contact YOUR-CONTACT
+snmp-server location YOUR-LOCATION
+```
+
+#### Wireless LAN Controller (WLC)
+1. Access the web admin page and log in
+2. If you are running version 8.1 and later, on the new dashboard click "Advanced"
+3. Go to management Tab
+4. On SNMP sub-menu, select "Communities"
+5. Click "New..."
+6. Add your community name and leave IP addresses empty
+7. Click Apply and Save
+
+### Infoblox
+#### NIOS 7.x
+1. Access the web admin page and log in
+2. Go to Grid tab > Grid Manager
+3. In the right menu select "Grid properties"
+4. Select "SNMP" menu
+5. Click "Enable SNMPv1/SNMPv2 Queries"
+6. Add your community
+7. Click Save & Close
+
+### Juniper
+#### Junos OS
+```
+set snmp description description
+set snmp location location
+set snmp contact contact
+set snmp community YOUR-COMMUNITY authorization read-only
+```
+
+### Palo Alto
+#### PANOS 6.x/7.x
+1. Access the web admin page and log in
+2. Go to Device tab > Setup
+3. Go to the sub-tab "Operations"
+4. Click "SNMP Setup"	
+5. Enter your SNMP community and then click "OK"
+6. Click Apply
+
+Note that you need to allow SNMP on the needed interfaces. To do that you need to create a network "Interface Mgmt" profile for standard interface and allow SNMP under "Device > Management > Management Interface Settings" for out of band management interface.
+
+
+## Operating systems
+### Linux (snmpd)
+
+Replace your snmpd.conf file by the example below and edit it with appropriate community in "RANDOMSTRINGGOESHERE".
+
+```
+vi /etc/snmp/snmpd.conf
+```
+
+```
```

Comment:
Possibly point to the file in git so we don't miss this if the file needs to be updated.  Anyone agree/disagree?